core-common/src/test/kotlin/com/hedvig/android/core/JSONObjectTest.kt

```diff
@@ -8,115 +8,115 @@ import org.json.JSONObject
 import org.junit.Test
 
 class JSONObjectTest {
```

Comment:
Oops did I fail to format this somehow? That's interesting, gotta check if there's some issue with which files ktlint is looking at to format.

Reply:
Addressed here https://github.com/HedvigInsurance/android/pull/1118